src/backend/storeManagers/legendary/games.ts

```diff
@@ -785,7 +787,8 @@ export async function importGame(
 
   const res = await runLegendaryCommand(command, { abortId: appName })
   addShortcuts(appName)
-
+  const stdLogs = convertStdLogsIntoObj(res.stderr)
+  res.error = stdLogs.ERROR.join('')
```

Comment:
I think the convertStdLogsIntoObj might be a bit too much for what we need here, in other places we simply check if the output matches `ERROR` like in https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/25247219fbc38547675990ebdc07df8317a84d85/src/backend/storeManagers/nile/games.ts#L134

it should make the code simpler without the need of this new object